Network/RSA/RSAConnection.cs

```diff
@@ -128,20 +126,24 @@ public bool IsRSACommunicationActive
             private set => isRSACommunicationActive = value;
         }
 
+        #endregion Properties
+
+        #region Methods
+
         /// <summary>
-        /// Initializes the RSA communication data.
+        /// Initializes the RSA communication serialisedPacket.
         /// Sends our information to the communication partner.
         /// Subscribes to the RSA packet events.
         /// </summary>
         private async void ExchangePublicKeys()
         {
             Connection.RegisterStaticPacketHandler<RSAKeyInformationRequest>((rsaKeyRequest, connection) =>
             {
-                connection.UnRegisterStaticPacketHandler<RSAKeyInformationRequest>();
+                connection.DeregisterStaticPacketHandler<RSAKeyInformationRequest>();
```

Comment:
Change "Deregister" back to UnRegister. That is one of the most used methods and u did rename it. That is a no go.
Also I want to stick to "UnRegister", so please do not mark it as obsolete :)